scripts/privacyidea

```diff
@@ -988,7 +988,7 @@ def create_arguments():
                           "of length",
                           metavar="NUMBER",
                           type=int,
-                          default=0)
+                          default=6)
```

Comment:
Cornelius Kölbel <notifications@github.com> writes:

>> @@ -988,7 +988,7 @@ def create_arguments():
>>                            "of length",
>>                            metavar="NUMBER",
>>                            type=int,
>> -                          default=0)
>> +                          default=6)
>
> If you revert this to default=0 I can merge this.

Hopefully done.

Jochen

-- 
The only problem with troubleshooting is that the trouble shoots back.